strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/domain/ArtifactEntry.java

```diff
@@ -38,6 +40,8 @@
     @ManyToMany(targetEntity = ArtifactTagEntry.class)
     private Set<ArtifactTag> tagSet;
 
+    private Map<String, String> checkSumsMap;
```

Comment:
`checkSumsMap` I think that `checksums` as a name would be enough

Reply:
+1

Reply:
Understood, I will implement this change as soon as I figure out another thing about this issue (how to test it). Then I will put the whole le in a pull request update